### PR TITLE
added a tip to register annotations namespaces

### PR DESCRIPTION
**Pull request by wolfwolker**

### CODE REVIEW

**Comment by xabbuh** [comment]
I am not against adding such a tip. But I suggest that we check all other places where we document the use of annotations in the components section and write a generic tip that we could include everywhere.

**Review by roman-1983 — APPROVED** [review]

**Comment by HeahDude** [comment]
Hello @wolfwolker, would you like to address @xabbuh's comment?

**Comment by wolfwolker** [comment]
Thanks for remember it to me @HeahDude. Yes I totally agree with @xabbuh, but I've been very busy these past weeks. I'm gonna try to add the tip also to other components sections in the following days.

**Comment by HeahDude** [comment]
ping @wolfwolker :) Are you still interested in finishing this PR?

**Comment by wolfwolker** [comment]
ups, totally forgot about it. I'll address it for this week. My apologies @HeahDude

**Comment by HeahDude** [comment]
No worries :), thanks!

**Comment by wolfwolker** [comment]
well, It's done. I created a snippet file under _includes directory with the tips and included it in three places I found with annotations loader usage.

But I'm not sure if it is working, how can I see these rst files with the symfony docs web look and feel?

File: components/serializer.rst [hunk 1]
```diff
@@ -314,6 +314,8 @@ You are now able to serialize only attributes in the groups you want::
 
 .. _ignoring-attributes-when-serializing:
 
+.. include:: /_includes/_rewrite_rule_tip.rst.inc
```
Comment (xabbuh) [inline components/serializer.rst]: I guess this should be moved before the previous label.

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
@xabbuh I did all the changes you asked for. We should check if this note is still relevant for newer Symfony versions. Thanks!

**Review by javiereguiluz — APPROVED** [review]

**Review by javiereguiluz — APPROVED** [review]

**Comment by javiereguiluz** [comment]
@wolfwolker it took us a long time to merge your contribution, but we finally did. Thank you and congrats on your first Symfony Docs contribution!

**Comment by wolfwolker** [comment]
thks @javiereguiluz ;)

**Pull request merged**